### DISCUSSION TITLE
The README looks so scuffed

### DISCUSSION THREAD

**Discussion by afkvido** [Prodigy-Hacking/ProdigyMathGameHacking#1301] [(C) Suggestions]
[This is the README at time of making this post](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/blob/c79d9b92b392b137c2683d98ab185cd9232716ba/README.md)

Now..... it looks weird.

The PMGH logo, and asking for stars just looks like a tower, and I think it's quite messy, especially in comparison to the rest of the README which is **super clean**.

###### PS, yama what the fuck is that commit message.... https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/commit/c79d9b92b392b137c2683d98ab185cd9232716ba

**Comment by afkvido** [comment]
i was gonna make a PR that cleaned up the readme but I don't want 32n to burn down my house

**Comment by donaldli2020** [comment]
> The PMGH logo, and asking for stars just looks like a tower, and I think it's quite messy, especially in comparison to the rest of the README which is **super clean**.

yeah, this part looks weird

**Comment by LeoBadeaux** [comment]
I've recently made changes to the README that improve the style a bit. The readme looks great now.

**Reply by afkvido** [reply]
thanks!

**Reply by afkvido** [reply]
WAIT ITS BROKEN

**Reply by afkvido** [reply]
one of the shields is broken

**Reply by LeoBadeaux** [reply]
> one of the shields is broken

I've noticed, it's just a glitch project that's going inactive and rebooting. I didn't really want to remove it.
